src/components/mypage/ReviewItem.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import Dot from './Dot';
+import { RATING_TEXT } from '@/constants/place';
+import { ReviewItem } from '@/types/mypage';
+import { format } from 'date-fns';
+import Image from 'next/image';
+
+export default function ReviewItem({ data, isLast }: { data: ReviewItem; isLast: boolean }) {
+  const { id, rating, createdAt, content } = data;
+  const date = format(new Date(createdAt), 'yyyy.MM.dd');
```

Comment:
디자인 상으로는 `yy.MM.dd` 형식이라 이렇게 바꾸면 될 것 같아!
```suggestion
  const date = format(new Date(createdAt), 'yy.MM.dd');
```